### DISCUSSION TITLE
Fix your Readme.md

### DISCUSSION THREAD

**Discussion by aryan1403** [vincenzopalazzo/material-ui-swing#164] [Q&A]
Some label such as donation are not maintained properly

**Comment by aryan1403** [comment]
Nice Fixed Well

**Reply by vincenzopalazzo** [reply]
Hey @Hellboy-Aaryan, what is the problem with the readme?

**Reply by aryan1403** [reply]
@vincenzopalazzo I see that the donation icon was **not** visible to me other's were prefect but when I _refresh_ the page it was fine maybe some **glitch** was there _sorry_ for unrelated issue. If I find any issue I will inform you.
Thanks
